libs/forms/core/src/module.providers.ts

```diff
@@ -17,6 +21,8 @@ export interface DynModuleProviders {
   providers?: Provider[];
   validators?: DynValidatorProvider[];
   asyncValidators?: DynAsyncValidatorProvider[];
+  matchers?: DynControlMatcher[];
+  conditions?: DynControlCondition[];
```

Comment:
Facilitate the provision of matchers and conditions via `DynFormsModule.forFeature({ matchers, conditions })`